Sources/Logging/Logging.swift

```diff
@@ -27,14 +27,15 @@ import WASILibc
 /// A `Logger` is the central type in `SwiftLog`. Its central function is to emit log messages using one of the methods
```

Comment:
All the \`Logger\` references should be \`\`Logger\`\`?

Reply:
This I left purposefully like this since this IS the `Logger` docs, so I don't think we need to link TO the `Logger` 🤔